kuma/wiki/apps.py

```diff
@@ -146,4 +152,4 @@ def on_document_spam_attempt_save(self, sender, instance, **kwargs):
         body = render_to_string('wiki/email/spam.ltxt',
                                 {'spam_attempt': instance})
         send_mail(subject, body, settings.DEFAULT_FROM_EMAIL,
-                  ['mdn-spam-watch@mozilla.com'])
+                  [config.EMAIL_LIST_FOR_FIRST_EDITS])
```

Comment:
Check in the prod constance admin if the default of the config value has been changed since it was deployed and change the value of the renamed config variable once deployed.